src/components/AltLabel.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { Typography } from "@material-ui/core";
+import { map } from "lodash";
+
+interface AltLabelProps {
+  altLabels?: { cs?: string }[];
+}
+
+const AltLabel: React.FC<AltLabelProps> = (props) => {
+  const altLabels = map(props.altLabels, "cs").join(", ");
+  if (!altLabels) return <></>;
```

Comment:
In cases like this it's probably better to return `null` than a `React.Fragment`, but no big deal.